xcdat/averager.py

```diff
@@ -20,275 +23,311 @@
 # Using xarrays data reduction functionality gives similar results, but is slower than the averager function defined here
 # Averaging axes are specified with a list (e.g., axis=['lat', 'lon']) rather than the CDAT convention (axis='xy')
 # It would be helpful to be able to get axis weights (get_axis_weights()) more generically (some information is hardcoded)
-    # If .bounds had a method to return a list of bounds entries, we could create weights for all dimensions
+# If .bounds had a method to return a list of bounds entries, we could create weights for all dimensions
 # May want to keep the axis weights or full weight array (i.e., add this information into the variable dataarray properties)
 # Is xvariable a good name?
 # Need to write unit tests
 # Can remove logging around bounds found in dataset
 # Functions are too specific to "lat" and "lon" (may have issues if dims are latitude / longitude)
 # Should remove these notes and outline...
 
+#: List of floats representing user-specified region axis bounds
+RegionAxisBounds = List[float]
```

Comment:
These are the accepted region axis bounds values (`lat_bounds` and `lon_bounds`). I think we should limit the scope of supported types to just a list of floats (either 1 value or 2 values [lower, upper]?)

Reply:
Fixed by limiting to a tuple of size two with validation function `_check_regional_bounds_input()`.